osu.Game/Online/API/DummyAPIAccess.cs

```diff
@@ -11,21 +11,45 @@ public class DummyAPIAccess : IAPIProvider
         public Bindable<User> LocalUser { get; } = new Bindable<User>(new User
         {
             Username = @"Dummy",
-            Id = 1,
+            Id = 1001,
```

Comment:
Surely you don't want this the default, otherwise the state is going to be online by default?

Reply:
It's what we expect in tests, yeah.